packages/pancake/src/files.js

```diff
@@ -118,6 +118,12 @@ const CreateDir = ( dir ) => {
  * @return {promise object}  - Boolean true for 👍 || string error for 👎
  */
 const WriteFile = ( location, content ) => {
+	/**
+	 * To compare file data hashes across platforms, we
+	 * need to remove \n or \r\n newlines
+	 */
+	content = content.replace( /[\r\n]/gm, '' );
```

Comment:
Can we keep the new line and get tests to pass for both platforms?  Or perhaps force the same new line across platforms